index.html

```diff
@@ -14,6 +14,18 @@ <h1>Documentation for Astarte</h1>
         <li><a href="astarte/latest/">Astarte</a></li>
         <li><a href="flow/latest/">Astarte Flow</a></li>
         <li><a href="astarte-kubernetes-operator/latest/">Astarte Operator</a></li>
+        <li>
+          Astarte Device SDKs
+          <ul>
+            <li><a href="device-sdks/elixir/latest/api/">Elixir</a></li>
+            <li><a href="device-sdks/esp32/latest/api/">ESP 32</a></li>
+            <li><a href="device-sdks/go/latest/api/">Go</a></li>
+            <li><a href="device-sdks/java/latest/api/">Java</a></li>
+            <li><a href="device-sdks/python/latest/api/">Python</a></li>
+            <li><a href="device-sdks/qt5/latest/api/">QT5 / C++</a></li>
+            <li><a href="device-sdks/rust/latest/api/">Rust</a></li>
```

Comment:
Currently, the docs contains only the esp32, python and java generated docs. If we want to use this file as a blueprint for the actual documentation, please remove the extra lines (e.g.: go, Qt5...)